packages/demo-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@modular-toolkit/demo-app",
-  "version": "3.1.5",
+  "version": "6.0.0-alpha.0",
```

Comment:
It seems releasing beta versions does not work correctly with Lerna 3, I've created an issue here: https://github.com/lerna/lerna/issues/1733